tools/core-build-tasks/src/tasks/index.ts

```diff
@@ -1,4 +1,13 @@
 export * from './api-extractor';
+export * from './bundle';
 export * from './clean';
+export * from './cleanCollateral';
+export * from './copy';
 export * from './coreLint';
+export * from './helpers';
 export * from './vitest';
+export * from './world';
+export * from './zip';
+
+export { generateContentsJsonTask } from './generateContentsJson';
+export type { GenerateContentsJsonParameters } from './generateContentsJson';
```

Comment:
Why not just an `export *` from `generateContentsJSON`?

Reply:
Also, you can combine into a single statement, not a huge deal to disambiguate the type export in these cases

Reply:
I didn't want to expose not needed functions. But, contents.json file is not required, even the addon validator is marking this file as not needed.
Then, removed this task and related files (test and static files).